scheduler/src/cook/mesos/api.clj

```diff
@@ -1093,7 +1093,8 @@
   {:job [s/Uuid]})
 
 (def UpdateRetriesRequest
-  {(s/optional-key :jobs) [s/Uuid]
+  {(s/optional-key :job) s/Uuid
```

Comment:
Is there a reason we want to support both `job` and `jobs`? Can't the same be achieved by passing a single job in `jobs` field?